diff --git a/cmake/ospray_options.cmake b/cmake/ospray_options.cmake
--- a/cmake/ospray_options.cmake
+++ b/cmake/ospray_options.cmake
@@ -12,7 +12,7 @@ set(OSPRAY_CMAKECONFIG_DIR
     "${CMAKE_INSTALL_LIBDIR}/cmake/ospray-${OSPRAY_VERSION}")
 
 set(ISPC_VERSION_REQUIRED 1.23.0)
-set(RKCOMMON_VERSION_REQUIRED 1.14.0)
+set(RKCOMMON_VERSION_REQUIRED 1.15.2)
 set(EMBREE_VERSION_REQUIRED 4.3.3)
 set(OPENVKL_VERSION_REQUIRED 2.0.1)
 set(OIDN_VERSION_REQUIRED 2.3.0)
